pkg/connector/iam_user.go

```diff
@@ -111,6 +125,63 @@ func iamUserProfile(ctx context.Context, user iamTypes.User) map[string]interfac
 	return profile
 }
 
+func getLastLogin(ctx context.Context, client *iam.Client, user iamTypes.User) *time.Time {
+	logger := ctxzap.Extract(ctx).With(
+		zap.String("user_id", *user.UserId),
+	)
+
+	res, err := client.ListAccessKeys(ctx, &iam.ListAccessKeysInput{UserName: user.UserName})
+	if err != nil {
+		logger.Error("Error listing access keys", zap.Error(err))
```

Comment:
If there's an error listing access keys, should we fall back to using the password last used time instead of returning nil?